api/__tests__/ministry.spec.ts

```diff
@@ -0,0 +1,94 @@
+//
+// Copyright © 2020 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import { Response } from 'express';
+import fs from 'fs';
+import { camelCase } from 'lodash';
+import path from 'path';
+import { Pool } from 'pg';
+import { fetchMinistrySponsors } from '../src/controllers/ministry';
+
+const p0 = path.join(__dirname, 'fixtures/select-ministry-sponsors.json');
+const selectMinistrySponsors = JSON.parse(fs.readFileSync(p0, 'utf8'));
+
+const client = new Pool().connect();
+
+jest.mock('../src/db/utils', () => ({
+    generateNamespacePrefix: jest.fn().mockReturnValue('c8c7e6'),
+    transformKeysToCamelCase: jest.fn().mockImplementation(data => {
+        const obj = {};
+        Object.keys(data).forEach(key => {
+            obj[camelCase(key)] = data[key];
+        });
+
+        return obj;
+    }),
+}));
+
+class FauxExpress {
```

Comment:
It might be a good opportunity to create `__tests__/src/fauxexpress.ts` and in it `export default class FauxExpress` then import it in each of the test suites that use it. It'll save a little code duplication.

Reply:
Have implemented this change. Do you want to quickly check it out as a final review?

Reply:
Nope. If the tests run I'm happy.